Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN apt-get install -y build-essential \
     python-zmq
 
 RUN pip install py4j \
-    ipython[notebook] \
+    ipython[notebook]==3.2 \
```

Comment:
Check on this one, unless I'm misunderstanding what he's saying:

https://github.com/jupyter/notebook/issues/309